integrated_channels/canvas/client.py

```diff
@@ -76,7 +76,13 @@ def create_content_metadata(self, serialized_data):
         """
         self._create_session()
 
-        course_details = json.loads(serialized_data.decode('utf-8'))['course']
+        desired_payload = json.loads(serialized_data.decode('utf-8'))
+        course_details = desired_payload['course']
+        # this makes the course use a 'participation' type of 'Course' instead of the default 'Term'
+        # which allows us to correctly tell Canvas to honor start/end dates instead of using the
+        # one from the default Term Canvas may end up using for this course
+        desired_payload['course']['restrict_enrollments_to_course_dates'] = True
```

Comment:
can this be added to the canvas content exporter instead of altering the data in the client?